### PR TITLE
add release page

### PR DESCRIPTION
**Pull request by SamYuan1990**
Signed-off-by: SamYuan1990 <yy19902439@126.com>

### CODE REVIEW

**Comment by SamYuan1990** [comment]
@davidkhala , @xiaohui249 
for this pr, I suggestion we have a meeting as working remote together to complete it.

File: build.gradle [hunk 1]
```diff
@@ -30,6 +31,23 @@ task checkstyle(type: Checkstyle) {
     classpath = files()
 }
 
+publishing {
+    repositories {
+        maven {
+            name = "GitHubPackages"
+            url = uri("https://maven.pkg.github.com/SamYuan1990/java-gm")
```
Comment (davidkhala) [inline build.gradle:38]: 临时放下？

Comment (SamYuan1990) [reply build.gradle:38]: 不是不可以。但是会麻烦些。

**Review by davidkhala — APPROVED** [review]
TODO @davidkhala 
David should align variable name with AZP Library Variables

**Pull request merged**